tokio-stream/src/wrappers/watch.rs

```diff
@@ -66,6 +67,13 @@ impl<T: 'static + Clone + Send + Sync> WatchStream<T> {
             inner: ReusableBoxFuture::new(async move { (Ok(()), rx) }),
         }
     }
+
+    /// Create a new `WatchStream` that waits for the value to be changed.
+    pub fn new_on_changed(rx: Receiver<T>) -> Self {
```

Comment:
I would like to see at least one example using this method in the documentation.

Reply:
done